frontend/src/main/java/puf/frisbee/frontend/viewmodel/GameViewModel.java

```diff
@@ -309,33 +322,55 @@ public BooleanProperty getQuitConfirmDialogProperty() {
 	public void showQuitConfirmDialog() {
 		if (!this.showGameOverDialog.getValue()) {
 			this.timeline.pause();
+			this.gameModel.setCurrentCountdown(this.second);
 			this.showQuitConfirmDialog.setValue(true);
 		}
 	}
+
+	public void saveGame() {
+		this.teamModel.setTeamLevel(this.levelModel.getCurrentLevel());
+		this.teamModel.setTeamScore(this.labelScore.getValue());
+		this.teamModel.setTeamLives(this.remainingLives);
+		this.gameModel.setCurrentCountdown(this.gameModel.getInitialCountdown());
+	}
+
+	public void continueAfterLevelSucceeded() {
+		this.levelModel.incrementCurrentLevel();
+		// TODO: save current lives, score and level of team to backend later on
+		this.saveGame();
+	}
+
+	public void pauseAfterLevelSucceeded() {
+		this.levelModel.setCurrentLevel(this.levelModel.getCurrentLevel() + 1);
```

Comment:
Seh ich grad schon nichts mehr, oder ist das nicht das gleiche wie `this.levelModel.incrementCurrentLevel();` (und damit das gleiche wie `continueAfterLevelSucceeded()`)?

Reply:
Gut gesehen. Habe auch noch zwei andere ganz ähnliche Dinge gleich mitrepariert … siehe nächster Commit/Push.